### PR TITLE
Updated Leaderboard styles at StandUp

### PR DESCRIPTION
**Pull request by ashtilawat23**
Leaderboard styles changed at meeting

### CODE REVIEW

File: src/components/pages/Leaderboard/Leaderboard.css [hunk 1]
```diff
@@ -19,16 +18,16 @@ h2 {
 
 ul {
     list-style-type: none;
-    margin-top: 1.5%;
+    margin-top: 2.8%;
     display: flex;
     flex-direction: column;
     align-items: center;
+    padding: 0;
 }
```
Comment (L-Steinmacher) [inline src/components/pages/Leaderboard/Leaderboard.css:27]: Nothing major but double space here.

File: src/components/pages/Leaderboard/Leaderboard.css [hunk 2]
```diff
@@ -80,17 +81,18 @@ ul {
     color: black; 
     margin-bottom: -1px;
     padding: 0.5rem 0.75rem;
-    border-bottom: 4px double rgba(0, 129, 198, 1);
-    box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24);
+    
+    /* box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.24); */
     transition: all 0.3s cubic-bezier(.25,.8,.25,1);
   }
```
Comment (L-Steinmacher) [inline src/components/pages/Leaderboard/Leaderboard.css:88]: Remove commented out/ unused code and remove empty spaces.

**Review by L-Steinmacher — APPROVED** [review]
Other than a few spacing issues and removal of commented out css the code looks good!

**Review by eet349 — APPROVED** [review]
Looks like you made those changes we discussed with Anthony. Looks good.

**Pull request merged**